target/linux/ramips/dts/APAC1200.dts

```diff
@@ -0,0 +1,174 @@
+ *  LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ *  A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ *  HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ *  SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ *  LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ *  DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ *  THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ *  (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ *  OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+/dts-v1/;
+
+#include "mt7620a.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "V7,apac1200", "ralink,mt7620a-soc";
+	model = "V7 APAC1200";
+
+	chosen {
+		bootargs = "console=ttyS0,57600";
+	};
+
+	gpio-keys-polled {
+		compatible = "gpio-keys";
+		#address-cells = <1>;
+		#size-cells = <0>;
+		poll-interval = <20>;
+
+		reset {
+			label = "reset";
+			gpios = <&gpio0 2 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+	};
+
+	gpio-leds {
+		compatible = "gpio-leds";
+
+		wlan2g {
+			label = "apac1200:blue:wlan2g";
+			gpios = <&gpio0 1 GPIO_ACTIVE_HIGH>;
+		};
+
+		wlan5g {
+			label = "apac1200:yellow:wlan5g";
+			gpios = <&gpio0 12 GPIO_ACTIVE_HIGH>;
+		};
+	};
+};
+
+&spi0 {
+	status = "okay";
+
+	m25p80@0 {
+		#address-cells = <1>;
+		#size-cells = <1>;
+		compatible = "jedec,spi-nor";
+		reg = <0>;
+		spi-max-frequency = <10000000>;
+
+		partition@0 {
+			label = "u-boot";
+			reg = <0x0 0x30000>;
+			read-only;
+		};
+
+		config: partition@30000 {
+			label = "config";
+			reg = <0x30000 0x10000>;
+			read-only;
+		};
+
+		factory: partition@40000 {
+			label = "factory";
+			reg = <0x40000 0x10000>;
+			read-only;
+		};
+
+		partition@50000 {
+			label = "firmware";
+			reg = <0x50000 0x7b0000>;
+		};
+	};
+};
+
+&pinctrl {
+        state_default: pinctrl0 {
+                gpio {
+                        ralink,group = "i2c", "uartf";
+                        ralink,function = "gpio";
+                };
+        };
+};
+
+
+&ethernet {
+        mtd-mac-address = <&config 0x29>;
+        mtd-mac-address-increment = <3>;
+        status = "okay";
+        pinctrl-names = "default";
+        pinctrl-0 = <&rgmii1_pins &rgmii2_pins &mdio_pins>;
+        mediatek,portmap = "wllll";
+
+        port@4 {
+                status = "okay";
+                phy-mode = "rgmii";
+                phy-handle = <&phy4>;
+        };
+
+        mdio-bus {
+                status = "okay";
+
+                phy4: ethernet-phy@4 {
+                        reg = <4>;
+                        phy-mode = "rgmii";
+                };
+
+        };
+};
+
+&gsw {
+	mediatek,port4 = "gmac";
+	mtd-mac-address = <&config 0x29>;
+};
+
+
+&pcie {
+	status = "okay";
+
+	pcie-bridge {
+		mt76@0,0 {
```

Comment:
I'm confused. According to your commit message you have a QCA988X (presumably attached via PCI) and the in-SoC mt7620 wireless.

But this whole node is about a MT76x2e, MT7603, MT7628 and MT7688 and you don't even include the corresponding mt76 driver in your image. Copy/Paste?